test/route.test.js

```diff
@@ -192,7 +192,7 @@ test('/v1/foobar should redirect to ./foobar/static/index.html - in plugin', asy
   t.plan(3)
   const fastify = Fastify()
 
-  fastify.register(async function (fastify, options) {
+  await fastify.register(async function (fastify, options) {
```

Comment:
This is not needed, beucase .inject() would register all plugins.

Reply:
```suggestion
  fastify.register(async function (fastify, options) {
```